transform-plugins/src/main/java/co/cask/hydrator/plugin/ValueMapper.java

```diff
@@ -83,25 +84,39 @@ public Config(String mapping, String defaults) {
      * subsequent methods.
      */
     private void parseConfiguration() {
-      Map<String, String> defaultsMapping = new HashMap<>();
       if (!defaults.isEmpty()) {
         String[] defaultsList = this.defaults.split(",");
         for (String defaultValue : defaultsList) {
           String[] defaultsArray = defaultValue.split(":");
-          defaultsMapping.put(defaultsArray[0], defaultsArray[1]);
+          if (defaultsArray.length != 2) {
+            throw new IllegalArgumentException("Defaults should contain source field and its corresponding default " +
+                                                 "value in the format: " +
+                                                 "<source-field>:<default-value>[,<source-field>:<default-value>]*" +
+                                                 "For eg: lang_code:English,country_code:Britain");
```

Comment:
Lets include the problematic default. "Invalid default " + defaultValue + ". Defaults should contain...."